### ISSUE TITLE
When picking an audio file that does not exist, the error message that appears is useless

### ISSUE THREAD

**Issue by xforever1313** [MeditationEnthusiasts/MeditationLogger#19]
The error message that appears when picking an audio file that does not exist on the start tab results in an error message that only contains the text in the field. It should include a message saying "File not found."

**Comment by xforever1313** [comment]
Revision: cf4357892345348cfc4d76a4f55aac065da56f7a
Author: Seth Hendrick <seth@shendrick.net>
Date: 1/11/2016 9:22:03 PM
Message:
Issue: 19

Fixed an issue where when we selected an audio file that doesn't exist, no error message appeared.

- If the field is blank, an "Audio file can not be empty." message appears.
- If a file can not be found, an "Could not open audio file: file" appears.
- Shows an error when an invalid file type is passed in.
----
Modified: MedEnthLogs/MedEnthDesktop/NAudioMusicManager.cs
Modified: MedEnthLogs/TestDesktop/NAudioMusicManagerTest.cs
